frontend/src/app/dora/types/reportResponse.ts

```diff
@@ -78,6 +78,23 @@ export interface FailureRate {
   changeFailureRateOfPipelines: FailureRateOfPipeline[];
 }
 
+export interface MeanTimeToRecoveryOfPipeline {
+  name: string;
+  step: string;
+  timeToRecovery: number;
+}
+
+export interface AvgMeanTimeToRecovery {
```

Comment:
`Mean` has the same meaning of `Avg`, it should be duplicated here. Similar naming issue for `avgMeanTimeToRecovery`.